collectors/pod.go

```diff
@@ -324,13 +338,22 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 				c.Name, nodeName)
 		}
 
+		if gpu, ok := req[v1.ResourceNvidiaGPU]; ok {
```

Comment:
@andyxning I have same question with you before. After some digging, I got resp when you get the pod from the apiserver, it will have request as below.
<img width="389" alt="2017-08-14 9 49 18" src="https://user-images.githubusercontent.com/2142864/29255671-0e9a94f0-80d6-11e7-9832-965e311b4ef0.png">
So for consistency I add gpu resource request record here